site-scrapers/ReverePopup/index.js

```diff
@@ -0,0 +1,65 @@
+// Unsure how often this clinic will pop up but seems to be weekly for now
+const { site } = require("./config");
+const moment = require("moment");
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${site.name} starting.`);
+    const info = await ScrapeWebsiteData(browser);
+    console.log(`${site.name} done.`);
+    return {
+        parentLocationName: "Revere Popup Clinic",
+        timestamp: moment().format(),
+        individualLocationData: [
+            {
+                ...site,
+                ...info,
+            },
```

Comment:
Looks like the `...site` is being overridden by `...info` which is getting the correct link in place.
The `parentLocation` should be the name of the scraper.  In this case `RevereCataldo`

Reply:
This is the clinic that is not run by Cataldo, it's run by the Revere Board of Health. Also, none of the properties in the site object are in the info object returned by ScrapeWebsiteData so I don't see how it could be overwritten? The spread operators should concatenate both objects so it'll look like 
`{
     name: "Revere Board of Health Popup Clinic",
    street: "140 American Legion Hwy",
    city: "Revere",
    zip: "02151",
    signUpLink: "https://www.maimmunizations.org//reg/4012583569",
    restrictions: "For those who live and work in Revere",
    extraData: "Moderna",
    hasAvailability: true,
    availability: {
        "04/30/21": {
            numberOfAvailableAppointments: 400,
           hasAvailability: true
        }
    }
}`

Reply:
Info and site are being concatenated by the spread operator and don't have any overlapping properties so that should not be the case. Doing a quick test, the return I got was:

{
  parentLocationName: 'Revere Popup Clinic',
  timestamp: '2021-04-29T13:25:59-04:00',
  individualLocationData: [
    {
      name: 'Revere Board of Health Popup Clinic',
      street: '140 American Legion Hwy',
      city: 'Revere',
      zip: '02151',
      signUpLink: 'https://www.maimmunizations.org//reg/4012583569',
      restrictions: 'For those who live and work in Revere',
      extraData: 'Moderna',
      hasAvailability: true,
      availability: [Object]
    }
  ]
}

Reply:
Also, the parent location for this scraper should not be RevereCataldo, this is a scraper for the clinic not run in collaboration with Cataldo, the other one that is a no-site-scraper is.

Reply:
I got something completely different in the `out_no_browser.json` file ...  **Prizer** not Moderna, **cic-health** not maimmunizations

```
{
  "version": 1,
  "timestamp": "2021-04-29T172945Z",
  "results": [
    {
      "hasAvailability": true,
      "availability": {
        "5/2/2021": {
          "numberAvailableAppointments": 872,
          "hasAvailability": true
        }
      },
      "name": "Rumney Marsh Academy - Cataldo Clinic",
      "street": "140 American Legion Hwy",
      "city": "Revere",
      "zip": "02151",
      "extraData": "Pfizer. Pre-registration tent will be on City Hall Lawn this week, Wed- Fri 11-3pm",
      "signUpLink": "https://www.cic-health.com/revere/rumneymarsh",
      "timestamp": "2021-04-29T13:29:44-04:00",
      "latitude": 42.4139504,
      "longitude": -71.0044181
    }
  ]
}
```